source/docs/beta/tasks/task-1-software-install.rst

```diff
@@ -0,0 +1,24 @@
+Task 1: Software Installation
+==================================
+
+Task: Choose, acquire, and install the software required for developing FRC software in the language you have been assigned (C++, Java or LabVIEW).
+
+Overview
+--------
+
+The FRC Control System can be programmed in LabVIEW, Java or C/C++. Teams should use the language they have been assigned for the Beta test. Documentation for installation can be found on the :ref:`docs/getting-started/getting-started-frc-control-system/index:Getting Started` page.
+
+.. note::
+   For this task, there is supplemental documentation provided in TeamForge for all languages. See the Documents and File Releases sections on TeamForge.
```

Comment:
```suggestion
```

Reply:
This is intentional and should not be removed. There is supplemental documentation in the closed TeamForge project which we will not post publicly such as links and serial numbers for beta software.